src/Users.php

```diff
@@ -42,7 +42,7 @@ public function __construct(Silex\Application $app)
         $this->authtokentable = $prefix . "authtoken";
         $this->users = array();
         $this->session = $app['session'];
-        $this->remoteIP = isset($_SERVER['REMOTE_ADDR']) ? $_SERVER['REMOTE_ADDR'] : "";
+        $this->remoteIP = $app['request']->getClientIp();
```

Comment:
Yeah nut user commands. We at least need to try/catch `app['request']` here and then use *localhost* or the like.